lib/config.js

```diff
@@ -47,10 +49,16 @@ class HomebaseConfig {
     // - can then be updated by APIs
     // - *may* be slightly massaged so long as it won't annoy users
     this.canonical = {}
+    this.diskUsage = null
+    this.estimatedDiskUsage = null
 
     if (configPath) {
       this.readFromFile(configPath)
     }
+
+    // align our estimated disk usage with real disk usage every 5 minutes
+    updateDiskUsage(this)
+    setInterval(updateDiskUsage.bind(this), 1000*60*5, this)
```

Comment:
I'd recommend calling `updateDiskUsage` and setting up the interval from `server.js`